cas/ctg_store.c

```diff
@@ -985,7 +985,8 @@ static int ctg_op_tick_ret(struct m0_ctg_op *ctg_op,
 
 	if (!op_is_active)
 		clink->cl_cb(clink);
-	m0_fom_phase_set(fom, next_state);
+	if (next_state >= 0)
```

Comment:
Please add a comment here to explain why.

Because we may pass -1 as the next phase in code code.

Reply:
ok